README.md

```diff
@@ -38,12 +38,14 @@ the demo requires the [React Native CLI](https://facebook.github.io/react-native
 $ npm install -g react-native-cli # install victory-native if you haven't already
 $ git clone https://github.com/FormidableLabs/victory-native
 $ cd victory-native
-$ npm install # install victory-native
-$ cd demo
-$ npm install # install the demo's dependencies, like react-native
-$ react-native run-ios # or react-native run-android
+$ npm install   # install victory-native
+$ npm start     # start react-native packager manually before launching demo!
+$ npm demo:ios  # or react-native run-android
 ```
 
+Do **not** run `npm install` in the `demo/` directory, or the packager packager will crash due to
+"duplicate @providesModule declarations" found in `node_modules/` and `demo/node_modules`.
```

Comment:
i think they simply don't get installed, and are unneeded. i need to play around just a bit more, but i'll update the PR and remove `demo/package.json` if possible